README.md

```diff
@@ -1,2 +1,22 @@
 # FinalReport_filter
 Software to filter Illumina Genomestudio FinalReport files
+
+### Make virtual python enviroment
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip install --upgrade pip
+pip install -r requirements.txt
```

Comment:
I don't see any requirements.txt in your main/develop/feature. Did I miss it?

Reply:
Good point. Probably default mode in my mind. Removed.